config/manager/manager.yaml

```diff
@@ -49,6 +49,7 @@ spec:
             - "--logtostderr"
             - "--exempt-namespace=gatekeeper-system"
             - "--operation=webhook"
+          automountServiceAccountToken: true
```

Comment:
I think this field should be in the `spec` field of the pod template? Though I agree all-or-nothing for mounting the token is weird.

Reply:
Good catch.  Fixed.